### DISCUSSION TITLE
Streamlabs obs

### DISCUSSION THREAD

**Discussion by yeahbuddy2575** [fredemmott/streaming-remote#15] [Q&A]
Would this work with streamlabs? I see that streamlabs is an obs product so, in theory, it would work with it?

**Comment by fredemmott** [comment]
Streamlabs is forked from OBS; I gather some stuff works, some stuff doesn't. This plugin isn't tested with it, but it might 'just work' - but you'd probably need to rebuild it against whatever version of Qt streamlabs OBS is built with, rather than the one real OBS is built with.
